src/Type/Definition/Behavior/ArgumentsTrait.php

```diff
@@ -12,62 +12,38 @@
     /**
      * @var Argument[]
      */
-    private $args = [];
+    private $_arguments = [];
```

Comment:
Personally I don't think underscore is really necessary

Reply:
The underscore just signifies that it cannot be set directly from outside the class. This is more of a convention I came up with to maintain my sanity.